### DISCUSSION TITLE
Why share only full drive, not a directory?

### DISCUSSION THREAD

**Discussion by HubKing** [neutrinolabs/xrdp#2261] [Q&A] [answered]
This is not a question specifically about XRDP, but about RDP itself. It seems that I can share a full drive with the remote. But in my opinion, people would be more interested in sharing a directory rather than a full drive. So, why does RDP only allow a drive, not a directory?

**Comment by bmullan** [comment, marked as answer]
@HubKing 

I use xfreerdp to access Ubuntu linux desktop systems that have xrdp installed and I know at least that lets you specify just a directory if thats all I want. 

**_Example:_**

> _$ xfreerdp /cert-ignore **/drive:my-shared_dir,/home/bmullan/my-shared_dir** /v:IP-of-ubuntu-system_

In the above when I log into that ubuntu/xrdp installed system my local directory named "my-shared_dir" is shared with the remote ubuntu system and I can copy files to/from that directory.   I'm not sharing the whole "local" drive.

brian

**Reply by HubKing** [reply]
On Windows, only a drive is selectable. So, I guess that it is not in the protocol, but just the MS client's limitation.

https://cat.pdx.edu/wp-content/uploads/2020/05/RDP_Folder_Sharing_Windows_2-400x333.png

**Reply by bmullan** [reply]
Freerdp is available for Windows, Mac - Linux and I've seen lots of windows users use Freerdp for their RDP connectivity needs.

Look here to get the Windows version:

https://github.com/FreeRDP/FreeRDP/wiki/PreBuilds
